### ISSUE TITLE
Python Error on write_html.py

### ISSUE THREAD

**Issue by scriptzteam** [libertysoft3/reddit-html-archiver#1]
```
./write_html.py
Traceback (most recent call last):
  File "./write_html.py", line 750, in <module>
    generate_html(args.min_score, args.min_comments, hide_deleted_comments)
  File "./write_html.py", line 97, in generate_html
    subs = get_subs()
  File "./write_html.py", line 619, in get_subs
    return [d.name for d in os.scandir('data') if d.is_dir()]
AttributeError: 'module' object has no attribute 'scandir'
```

**Comment by libertysoft3** [comment]
@scriptzteam I just pushed a fix, can you try again? I think I was accidentally requiring python 3.5+.

**Comment by libertysoft3** [comment]
fixed in 8385bb6f66893c8032af294180fb162b68ec7c68
